### ISSUE TITLE
Inconsistent adjective-like nouns markings

### ISSUE THREAD

**Issue by The-Keeper** [UniversalDependencies/UD_Russian-PUD#4]
```
# newdoc id = w05006
# sent_id = w05006024
# text = Испанский Золотого века или ранний современный испанский является вариантом языка, составляющим переход от средневекового испанского к современному испанскому.
# english_text = Golden Age Spanish or Early Modern Spanish is the variant of the language that constitutes the transition from Medieval Spanish to Modern Spanish.
1	Испанский	испанский	ADJ	JJ	Case=Nom|Degree=Pos|Gender=Masc|Number=Sing	8	nsubj	_	_
2	Золотого	золотой	ADJ	JJ	Case=Gen|Degree=Pos|Gender=Masc|Number=Sing	3	amod	_	Proper=True
3	века	век	NOUN	NN	Animacy=Inan|Case=Gen|Gender=Masc|Number=Sing	1	nmod	_	_
4	или	или	CCONJ	CC	_	7	cc	_	_
5	ранний	ранний	ADJ	JJ	Case=Nom|Degree=Pos|Gender=Masc|Number=Sing	7	amod	_	_
6	современный	современный	ADJ	JJ	Case=Nom|Degree=Pos|Gender=Masc|Number=Sing	7	amod	_	_
7	испанский	испанский	ADJ	JJ	Case=Nom|Degree=Pos|Gender=Masc|Number=Sing	1	conj	_	_
8	является	являться	VERB	VBC	Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Mid	0	root	_	_
9	вариантом	вариант	NOUN	NN	Animacy=Inan|Case=Ins|Gender=Masc|Number=Sing	8	xcomp	_	_
10	языка	язык	NOUN	NN	Animacy=Inan|Case=Gen|Gender=Masc|Number=Sing	9	nmod	_	SpaceAfter=No
11	,	,	PUNCT	,	_	12	punct	_	_
12	составляющим	составлять	VERB	VBNL	Aspect=Imp|Case=Ins|Gender=Masc|Number=Sing|Tense=Pres|VerbForm=Part|Voice=Act	9	acl	_	_
13	переход	переход	NOUN	NN	Animacy=Inan|Case=Acc|Gender=Masc|Number=Sing	12	obj	_	_
14	от	от	ADP	IN	_	16	case	_	_
15	средневекового	средневековый	ADJ	JJ	Case=Gen|Degree=Pos|Gender=Masc|Number=Sing	16	amod	_	_
16	испанского	испанский	ADJ	JJ	Case=Gen|Degree=Pos|Gender=Masc|Number=Sing	13	nmod	_	_
17	к	к	ADP	IN	_	19	case	_	_
18	современному	современный	ADJ	JJ	Case=Dat|Degree=Pos|Gender=Masc|Number=Sing	19	amod	_	_
19	испанскому	испанский	NOUN	NN	Animacy=Inan|Case=Dat|Gender=Masc|Number=Sing	13	nmod	_	SpaceAfter=No
20	.	.	PUNCT	.	_	8	punct	_	_
```

Here lemmas "испанский" at positions 1, 7 and 16 are makred as ADJ and at 19 as NOUN. All those entries actually mean "Spanish language" (similar to "Spanish" on its own being short for "Spanish language") and function as nouns in this sentence. The tricky thing is that, for example, in "испанский язык" (Spanish language) "испанский" would be an adjective modifying the noun "язык". 
Context usually leaves no ambiguity because whenever "испанский" or similar "X-ский" word happens to be on its own, it means "X-ish language" but with an acomppaying noun it is an adjective that means "related to X", so my bet is that all of instances of "испанский" here should be marked as nouns.

Same thing for "испанский" in # sent_id = w05006058   and "финский" in  # sent_id = w01013096. 
Note that in # sent_id = w01013083   "датский" at (7) should be a NOUN while "исладнский" at (10) is an ADJ as it modifies the word "язык".

```
# sent_id = w01013102
# text = Чередование согласных является особенностью и финского, и северо-саамского диалектов, но не существует в южно-саамском, который имеет другую языковую историю.
# english_text = Consonant gradation is a feature in both Finnish and northern Sami dialects, but it is not present in south Sami, which is considered to have a different language history.
1	Чередование	чередование	NOUN	NN	Animacy=Inan|Case=Nom|Gender=Neut|Number=Sing	3	nsubj	_	_
2	согласных	согласный	NOUN	NN	Animacy=Inan|Case=Gen|Gender=Masc|Number=Plur	1	nmod	_	_
3	является	являться	VERB	VBC	Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Mid	0	root	_	_
4	особенностью	особенность	NOUN	NN	Animacy=Inan|Case=Ins|Gender=Fem|Number=Sing	3	xcomp	_	_
5	и	и	CCONJ	CC	_	6	cc	_	_
6	финского	финский	ADJ	JJ	Case=Gen|Degree=Pos|Gender=Masc|Number=Sing	10	amod	_	SpaceAfter=No
7	,	,	PUNCT	,	_	9	punct	_	_
8	и	и	CCONJ	CC	_	9	cc	_	_
9	северо-саамского	северо-саамский	ADJ	JJ	Case=Gen|Degree=Pos|Gender=Masc|Number=Sing	6	conj	_	_
10	диалектов	диалект	NOUN	NN	Animacy=Inan|Case=Gen|Gender=Masc|Number=Plur	4	nmod	_	SpaceAfter=No
11	,	,	PUNCT	,	_	14	punct	_	_
12	но	но	CCONJ	CC	_	14	cc	_	_
13	не	не	PART	NEG	Polarity=Neg	14	advmod	_	_
14	существует	существовать	VERB	VBC	Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act	3	conj	_	_
15	в	в	ADP	IN	_	16	case	_	_
16	южно-саамском	южный-саамский	ADJ	JJ	Case=Loc|Degree=Pos|Gender=Masc|Number=Sing	14	obl	_	SpaceAfter=No
17	,	,	PUNCT	,	_	19	punct	_	_
18	который	который	PRON	WDT	Case=Nom|Number=Sing	19	nsubj	_	_
19	имеет	иметь	VERB	VBC	Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act	16	acl:relcl	_	_
20	другую	другой	ADJ	JJ	Case=Acc|Degree=Pos|Gender=Fem|Number=Sing	22	amod	_	_
21	языковую	языковой	ADJ	JJ	Case=Acc|Degree=Pos|Gender=Fem|Number=Sing	22	amod	_	_
22	историю	история	NOUN	NN	Animacy=Inan|Case=Acc|Gender=Fem|Number=Sing	19	obj	_	SpaceAfter=No
23	.	.	PUNCT	.	_	3	punct	_	_
```

Note that here, (6) and (9) are adjectives as they modify (10). However (16) could be interpreted as an adjective whose noun was left out (as short for "южный-саамский диалект" to avoid reduplication) or as a noun "южно-саамский язык". The first interpretation reads as more likely one. In the original sentence the compound "south Sami" reads as a noun (and less likely as an adjective) so the ambiguity is both on the part the original sentence and the translation. One could argue that [broader context](https://en.wikipedia.org/wiki/Scandinavia#Sami_languages) indicates that there should be a noun, however interpretation doesn't make a big difference semantically.

One could also argue that since treebanks contain adjectival sense of those short-for-language nouns much more often, they should be left as adjectives and let the parser decide the relationship between them. However there are other adjective-like nouns which are consistently marked as nouns when context makes meaning clear: "знакомый пейзаж" ("familiar.ADJ  scenery.NOUN") vs "мой знакомый" (my.DET acquaintance.NOUN)

**Comment by dan-zeman** [comment]
I would keep them all `ADJ` regardless whether the word _язык_ is present or not.

**Comment by Kira-D** [comment]
Fixed in 2.6 (f0fc882)
